pkg/database/realm.go

```diff
@@ -456,7 +462,14 @@ func (r *Realm) validateSMSTemplate(label, t string) {
 		r.AddError("smsTextTemplate", fmt.Sprintf("when expanded, the result message is too long (%v characters). The max expanded message is %v characters", l, SMSTemplateExpansionMax))
 		r.AddError(label, fmt.Sprintf("when expanded, the result message is too long (%v characters). The max expanded message is %v characters", l, SMSTemplateExpansionMax))
 	}
+}
 
+// enxRedirectDomain returns the configured ENX redirect domain for this realm.
+func (r *Realm) enxRedirectDomain() string {
```

Comment:
Small feature addition (package private) to support running more tests in parallel.